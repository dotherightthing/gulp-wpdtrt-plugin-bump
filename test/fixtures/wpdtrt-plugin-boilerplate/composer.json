{
    "name": "dotherightthing/wpdtrt-plugin-boilerplate",
    "autoload": {
        "psr-4": {
            "DoTheRightThing\\WPDTRT_Plugin_Boilerplate\\r_0_12_345\\": "src"
        }
    }
}
