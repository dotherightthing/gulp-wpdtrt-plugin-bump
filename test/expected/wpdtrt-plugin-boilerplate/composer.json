{
    "name": "dotherightthing/wpdtrt-plugin-boilerplate",
    "autoload": {
        "psr-4": {
            "DoTheRightThing\\WPDTRT_Plugin_Boilerplate\\r_1_0_0\\": "src"
        }
    }
}
